queries/prs_query.py

```diff
@@ -51,5 +64,22 @@ def prs_query(dbmc, repo):
     df_pr = df_pr.reset_index()
     df_pr.drop("index", axis=1, inplace=True)
 
+    # break apart returned data per repo
+    # and temporarily store in List to be
+    # stored in Redis.
+    pic = []
+    for i, r in enumerate(repos):
+        # convert series to a dataframe
+        c_df = pd.DataFrame(df_pr.loc[df_pr["id"] == r]).to_csv()
+
+        # add pickled dataframe to list of pickled objects
+        pic.append(c_df)
+
+    del df_pr
+
+    # store results in Redis
+    cm_o = cm()
+    ack = cm_o.setm(func=prs_query, repos=repos, datas=pic)
```

Comment:
whats ack? comment on all queries

Reply:
gotcha, 'ack' is a programming shorthand for 'acknowledgement' or 'acknowledged,' originating in embedded systems and networking. I used it reflexively but I'll change it.